test/integration/tasks.test.js

```diff
@@ -169,7 +176,7 @@ describe("Tasks", function () {
           }
           expect(res).to.have.status(200);
           expect(res.body).to.be.a("array");
-          expect(res.body[0].status).to.equal("completed");
+          expect(res.body[0].status).to.equal(TASK_STATUS.COMPLETED);
```

Comment:
Why aren't we destructuring `TASK_STATUS.COMPLETED` beforehand as already doing for other task status?

Reply:
👍 will destructure it.